components/core/src/clp/regex_utils/regex_translation_utils.cpp

```diff
@@ -92,6 +93,14 @@ using StateTransitionFuncSig
  */
 [[nodiscard]] StateTransitionFuncSig dot_state_transition;
 
+/**
+ * Appends regex metacharacters literally to the wildcard string.
```

Comment:
Lets re-word this a bit:

"Appends a regex meta character as a literal character to the wildcard string."